src/service/variable-source.js

```diff
@@ -236,11 +236,13 @@ export class VariableSource {
     keys.sort((s1, s2) => s2.length - s1.length);
     const all = keys.join('|');
     // Match the given replacement patterns, as well as optionally
-    // arguments to the replacement behind it in parantheses.
+    // arguments to the replacement behind it in parentheses.
     // Example string that match
     // FOO_BAR
     // FOO_BAR(arg1)
     // FOO_BAR(arg1,arg2)
-    return new RegExp('\\$?(' + all + ')(?:\\(([0-9a-zA-Z-_.,]+)\\))?', 'g');
+    // FOO_BAR(arg1, arg2)
+    return new RegExp('\\$?(' + all + ')' +
+        '(?:\\(([0-9a-zA-Z-_.]+(?:,\\s*[0-9a-zA-Z-_.]*)*)\\))?', 'g');
```

Comment:
Could have just done:
```
return new RegExp('\\$?(' + all + ')(?:\\(([0-9a-zA-Z-_.,\\s]+)\\))?', 'g');
```
(added the "\\s") but did not want to allow FOO_BAR(a r g 1, a r g 2)

Reply:
Nit: We can simplify by adding a negative lookahead for `,`:

```js
/\((\s*(?!,)(?:\s*,?\s*[0-9a-zA-Z-_.]+)*)\)/
```

Reply:
Cool, I didn't know about negative lookaheads. Will do.